ppr-ui/src/components/mhrRegistration/HomeLocation/HomeCivicAddress.vue

```diff
@@ -9,14 +9,13 @@
           <div class="form__row">
             <v-text-field
               autocomplete="new-password"
-              id="street"
+              :id="streetId"
```

Comment:
is `streetId` a dynamic property? If so, where is this defined?

If not a dynamic property (ie a string), you don't need the dynamic prop binding `:`

Reply:
Good question.. streetId is required by the Canada Post control... It must be defined as a dynamic property.. I tested both ways.. I mainly did this to prevent writing another wrapper similar to the baseAddress composable..